### PR TITLE
Default: New mese block texture

### PR DESCRIPTION
**Pull request by paramat**
Add missing texture credits for mese block, crystal and crystal fragment
/////////////////////////////////////////////////////////

meseeverind128

Identical to everamzah's submission here https://github.com/minetest/minetest_game/pull/1082#issuecomment-217928251 but with alpha channel removed and converted to indexed colours for a smaller file.
This is approved by myself and Ekdohibs, and +1'd by at least 12 others, which is rare for a texture PR.
Also added missing credits to VaneesaE for other mese textures.

### CODE REVIEW

**Comment by paramat** [comment]
Adding my approval since it's someone elses texture.

Approval by Ekdohibs http://irc.minetest.ru/minetest-dev/2016-05-16#i_4611789

**Comment by sofar** [comment]
:+1:

**Comment by PilzAdam** [comment]
Since there seem to be a dozen of these PRs now I'd like to repeat my rejection: I think that the word "MESE" should be readable on the texture.

**Comment by benrob0329** [comment]
@paramat +1

**Comment by paramat** [comment]
PilzAdam, it is, that's why everamzah's is an improvement on Ekdohibbs' version.
It is both mysterious looking symbols, necessary for an 'unknown/alien/ancient origin' while also looking like MESE and therefore explaining the name used to refer to it.

**Comment by Thomas--S** [comment]
May I ask why this texture is CC BY-SA now?
The first version of this texture was made by me and published under CC0.
Then @Ekdohibs improved it and published the change also under CC0.
@everamzah only changed the "M" and now it's CC BY-SA?

I don't have a problem with CC BY-SA, I'm just wondering.

**Comment by everamzah** [comment]
@Thomas--S indeed credit should go to you; it is correct I simply rotated and arranged the letters.

**Comment by Thomas--S** [comment]
@everamzah I don't think that credit should only go to me. I suggest the following:

```
Everamzah, Ekdohibs, paramat, Thomas-S (CC0)
  default_mese_block.png
```

All four people from my suggestion above contributed to the new texture. I would also prefer CC0, if this is ok for you (see above for details).

**Comment by everamzah** [comment]
>I would also prefer CC0, if this is ok for you...

Perfectly permissible.

**Comment by PilzAdam** [comment]
@paramat 

> necessary for an 'unknown/alien/ancient origin'

And why is it necessary to have an origin? In this whole discussion, everything seems to come down to the point that the Latin letters are not "alien enough". So my question to everyone who wants to change the texture is: what exactly is wrong if we leave the texture as it is currently and simply do not specify any origin or background story at all?

**Comment by benrob0329** [comment]
@PilzAdam why not have a backstory? We have a very strange block with very strange properties and no explanation for it, so why not add an explanation for it?

File: mods/default/README.txt [hunk 1]
```diff
@@ -185,6 +187,9 @@ asl97 (WTFPL):
 KevDoy (CC BY-SA 3.0)
   heart.png
 
+Everamzah (CC BY-SA 3.0)
```
Comment (Thomas--S) [inline mods/default/README.txt]: Please change this line according to this: https://github.com/minetest/minetest_game/pull/1093#issuecomment-219686804

**Comment by sofar** [comment]
> what exactly is wrong if we leave the texture as it is currently

It shouldn't have had readable letters in the first place:

- untranslatable
- stylistically out of place
- text is a reference to a computer program by made by Mircosotf, possibly trademarked

That last item already should be enough reason to remove all references to "mese" in the game immediately, with full support of every core dev on the team.

**Comment by PilzAdam** [comment]
@sofar 

> * untranslatable

"Mese" isn't a word with any meaning, so you don't have to translate it.

> * stylistically out of place

For me the "Mese" block is part of Minetest, since it was one of the first original blocks that existed. It is (for me) the definition of the "Minetest style".

> * text is a reference to a computer program by made by Mircosotf, possibly trademarked

WTF? Since when is a _reference_ to something trademarkable? Also, only some weird finish game devs refer to the MSN Messenger as "MESE", why would an American company care about that? This is ridiculous.

**Comment by paramat** [comment]
Sorry for not seeing the licence! will update that and credits.

**Comment by sfan5** [comment]
:-1: agree with @PilzAdam

**Comment by paramat** [comment]
Nore approves once licence is corrected http://irc.minetest.ru/minetest-dev/2016-05-17#i_4612540

**Comment by paramat** [comment]
With 2 disapprovals i think this needs rethinking, i might attempt a new texture.

**Comment by sofar** [comment]
@PilzAdam 

>> text is a reference to a computer program by made by Mircosotf, possibly trademarked

> WTF? Since when is a reference to something trademarkable? Also, only some weird finish game devs refer to the MSN Messenger as "MESE", why would an American company care about that? This is ridiculous.

It isn't ridiculous if this prevents people from using it in the USA for fear of legal action, which reduces freedom.

The argument that you don't believe this is irrelevant, what matters is that others can *potentially* see this as a problem.

I work for a US company, and it IS a potential problem for me.

**Comment by mtango688** [comment]
Wow. Just wow. Why shouldn't a texture of an ore have a non-ore image? Am I seeing things?

If consistency is a valued part of this project then the question should be why would you ever want an inconsistency?

If not wanting the player double-take at the sight of an ugly looking texture is a feature, then why was MESE ever not considered a place-holder in the first place?

**Comment by paramat** [comment]
Closed due to 2 disapprovals.

**Pull request closed without merging**